### PR TITLE
Move follow to src

### PR DESCRIPTION
**Pull request by zeroadam**
Move include/follow.php functions in to src/Model/Contact class.

Related to #3878

### CODE REVIEW

**Comment by zeroadam** [comment]
Running these changes for about an hour now. No major errors, and successfully connected with @Rudloff. Not sure what else to check to verify success.

File: src/Model/Contact.php [hunk 1]
```diff
@@ -1009,4 +1011,283 @@ public static function updateAvatar($avatar, $uid, $cid, $force = false)
...
+	 * $return['message'] error text if success is false.
+	 *
+	 * @brief Takes a $uid and a url/handle and adds a new contact
+	 * @param int    $uid
+	 * @param string $url
+	 * @param bool   $interactive
+	 * @param string $network
+	 * @return boolean|string
+	 */
+	public static function add($uid, $url, $interactive = false, $network = '')
```
Comment (MrPetovan) [inline src/Model/Contact.php]: Where `Contact::new` you renamed `new_contact` to?

Please rename it `Contact::create` instead.

File: src/Model/Contact.php [hunk 2]
```diff
@@ -1009,4 +1011,283 @@ public static function updateAvatar($avatar, $uid, $cid, $force = false)
 
 		return $data;
 	}
+
+	/**
+	 * @param integer $id contact id
+	 * @return boolean
+	 */
+	public static function update($id)
```
Comment (MrPetovan) [inline src/Model/Contact.php]: Please rename this function to `Contact::updateFromProbe` as it is unclear from the name and parameter alone.

Comment (zeroadam) [reply src/Model/Contact.php]: OK, i just saw it as "i'm updating a contact"

File: src/Model/Contact.php (hunk 1 above)
Comment (zeroadam) [reply src/Model/Contact.php]: yes, renamed new_contact to add. There was no new() function. Am I understanding correctly?

regardless, i will rename add to create and update calls.

Comment (MrPetovan) [reply src/Model/Contact.php]: The function calls currently are `Contact::new()` but the function you created is `Contact::add()`.

File: src/Model/Contact.php (hunk 2 above)
Comment (MrPetovan) [reply src/Model/Contact.php]: Usual `update` operation take an array of modified fields to update the according model/row. This is more like a self-update based on an URL probe.

File: src/Model/Contact.php [hunk 3]
```diff
@@ -1009,4 +1011,283 @@ public static function updateAvatar($avatar, $uid, $cid, $force = false)
...
+	 * $return['message'] error text if success is false.
+	 *
+	 * @brief Takes a $uid and a url/handle and adds a new contact
+	 * @param int    $uid
+	 * @param string $url
+	 * @param bool   $interactive
+	 * @param string $network
+	 * @return boolean|string
+	 */
+	public static function create($uid, $url, $interactive = false, $network = '')
```
Comment (MrPetovan) [inline src/Model/Contact.php]: I'm sorry, I didn't read carefully what the function was doing. Can you rename it to `createFromProbe` please? In the same spirit as `update`, I feel like the base  `create` function should take an array of fields to initialize a model/row with.

File: src/Model/Contact.php (hunk 1 above)
Comment (zeroadam) [reply src/Model/Contact.php]: Wow.. I didn't even see that!! You're right I'm amazed there were no errors, and I successfully connected with a new Friendica contact

File: src/Model/Contact.php (hunk 3 above)
Comment (zeroadam) [reply src/Model/Contact.php]: Sure! no worries

**Pull request merged**